ocp_install_env.sh

```diff
@@ -186,7 +223,7 @@ $(baremetal_network_configuration)
     clusterOSImage: http://$(wrap_if_ipv6 $MIRROR_IP)/images/${MACHINE_OS_IMAGE_NAME}?sha256=${MACHINE_OS_IMAGE_SHA256}
     apiVIP: ${API_VIP}
     ingressVIP: ${INGRESS_VIP}
-    dnsVIP: ${DNS_VIP}
+$(dnsvip)
```

Comment:
Might be safer to do the indentation here rather than in the function.

Reply:
Yeah I did think the same but all of the other functions adding install-config snippets work like this so I decided to align for consistency.